letsencrypt/continuity_auth.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, config, installer):  # pylint: disable=unused-argument
         :type installer: :class:`letsencrypt.interfaces.IInstaller`
 
         """
-        self.proof_of_pos = proof_of_possession.ProofOfPossession(installer)
+        self.proof_of_pos = proof_of_possession.ProofOfPossession(
+            installer, config)
```

Comment:
Nit: I'd flip the order so that `config` is the first argument and `installer` is the second. This is more consistent with the rest of the code base.